.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+**related issues** #{issue1}, #{issue2}
+
+---
+
+### description
+
+{describe your PR here}
+
+### checklist
+
+- [ ] I added the related issue(s) id in the related issues section (if any)
+  - if not, delete the related issues section
+- [ ] I described my changes and my decisions in the PR description
+- [ ] I read the development guidelines of the [CONTRIBUTING.md][development-guidelines]
+- [ ] the tests pass and have been updated if relevant
+- [ ] the code quality check pass
+
+❤️ Thank you for your contribution!
+
+[development-guidelines]: https://github.com/tournesol-app/tournesol/blob/main/CONTRIBUTING.md#development-guidelines
```

Comment:
I think it is ok if we put this link only once in the checklist

Reply:
Yes, that's what I did at the beginning but the size of the link was making the checklist slightly harder to read.